src/services/AbstractDFG.jl

```diff
@@ -723,17 +724,17 @@ the symbol of the relating factor. The first row and first column are factor and
 variable headings respectively.
 """
 function getAdjacencyMatrix(dfg::AbstractDFG; solvable::Int=0)::Matrix{Union{Nothing, Symbol}}
-    @warn "Deprecated function, please use getIncidenceMatrix as this will be removed in v0.6.1"
-    return getIncidenceMatrix(dfg, solvable)
+    error("Deprecated function, please use getBiadjacencyMatrix")
 end
 """
     $(SIGNATURES)
 Get a matrix indicating relationships between variables and factors. Rows are
 all factors, columns are all variables, and each cell contains either nothing or
 the symbol of the relating factor. The first row and first column are factor and
 variable headings respectively.
+Note: rather use getBiadjacencyMatrix
 """
-function getIncidenceMatrix(dfg::AbstractDFG; solvable::Int=0)::Matrix{Union{Nothing, Symbol}}
+function getAdjacencyMatrixSymbols(dfg::AbstractDFG; solvable::Int=0)::Matrix{Union{Nothing, Symbol}}
```

Comment:
We can keep it and rename it to something like this, but I don't think it should be used at all for calculations.

Reply:
Agreed, if it's not used and the sparse is equivalent, let's remove it.